src/staging/src/components/search/mainSearch.js

```diff
@@ -52,177 +43,49 @@ const MainSearch = ({ data: { activities, facilities } }) => {
-                        <i className="fas fa-search"></i>
-                      </InputAdornment>
-                    )
-                  }}
-                  value={searchText}
-                  onChange={event => {
-                    setSearchText(event.target.value)
-                  }}
-                  onKeyPress={ev => {
-                    if (ev.key === "Enter") {
-                      searchParkFilter()
-                      ev.preventDefault()
-                    }
-                  }}
-                />
-            </div>
-            <div className="col-3">
-              <Button
-                  variant="outlined"
-                  onClick={handleClickOpenFilter}
-                  className="bg-transparent rounded text-light mobile-filter mobile-search-element-height"
-              >
-                Filters
-              </Button>
-            </div>
-          </div>
-          <div className="row no-gutters pb-2 px-3">
-            <div className="col-12">
-              <Button
-                  variant="contained"
-                  fullWidth
-                  onClick={() => {
-                    handleClickOpenFilter()
-                    searchParkFilter()
-                  }}
-                  className="bcgov-normal-blue mobile-search-element-height"
-              >
-                Search
-              </Button>
-            </div>
-          </div>
-          <div className="row no-gutters px-3">
-            <div className="col-12">
-              <Button
-                  variant="contained"
-                  href="https://www.discovercamping.ca/"
-                  target="_blank"
-                  rel="norefferer"
-                  fullWidth
-                  onClick={() => {
-                    handleClickOpenFilter()
-                    searchParkFilter()
-                  }}
-                  className="bcgov-normal-gold mobile-search-element-height"
-              >
-                Book a campsite
-              </Button>
-            </div>
-          </div>
-        </div>
-      </div>
-    )
-  }
-
   return (
-    <div className="park-search-container park-search-text-container">
-      <div className="d-none d-lg-block v-align-abs">
-        {renderDesktop()}
+    <div className="parks-search-wrapper">
+      <div className="parks-search-header">
+        Find your next adventure
+      </div>
+      <div className="parks-search-field">
+        <InputLabel className="sr-only" htmlFor="park-search-text">
+            Search
+        </InputLabel>
+        <TextField
+          id="park-search-text"
+          variant="outlined"
+          placeholder="Plan your next adventure by searching for campsites and day use areas"
+          className="park-search-text-box h50p"
+          value={searchText}
+          onChange={event => {
+            setSearchText(event.target.value)
+          }}
+          onKeyPress={ev => {
```

Comment:
FYI if you wrap the fields in a form enter will submit:
```
<form onSubmit={(evt) => { evt.preventDefault(); searchParkFilter() }}>
</form>
```

I find that a bit simpler than treating having keypress and button submit events, but it looks like it would require some styling changes

Reply:
This was from the previous version, not attached to this method. However, would like to avoid the styling, so hanging off on changing it at this point.